lib/active_merchant/billing/gateways/ebanx.rb

```diff
@@ -149,9 +149,9 @@ def add_customer_responsible_person(post, payment, options)
         post[:payment][:person_type] = options[:person_type] if options[:person_type]
         if options[:person_type] && options[:person_type].downcase == 'business'
           post[:payment][:responsible] = {}
-          post[:payment][:responsible][:name] = customer_name(payment, options)
-          post[:payment][:responsible][:document] = options[:document] if options[:document]
-          post[:payment][:responsible][:birth_date] = options[:birth_date] if options[:birth_date]
+          post[:payment][:responsible][:name] = options[:responsible_name] if options[:responsible_name]
+          post[:payment][:responsible][:document] = options[:document] if options[:responsible_document]
```

Comment:
Should this be `= options[:responsible_document] if options[:responsible_document]`?

Reply:
Oof, good catch.

Reply:
Updated.